lib/response.js

```diff
@@ -822,6 +822,14 @@ res.get = function(field){
  */
 
 res.clearCookie = function clearCookie(name, options) {
+  if (options) {
```

Comment:
Sorry to make a comment on this again lol, take this as a nitpick, but if these checks were just made below line 833 they could assume `opts` and just be the individual if's.

Reply:
There will always be an expires option after the merge, either because options included it or because we set it if not.

we can check maxAge after the merge, because it should not be present regardless. I want the checks to be where they are so it is clear we are checking inputs. 

The perf impact here is very minimal, but requires some conditional checking regardless. 

Everyone pays the cost of the first check, but only some pay the (extremely minimal) cost of the further two.